.github/scripts/otp-compliance.es

```diff
@@ -440,8 +440,9 @@ fix_name(Name, Sbom) ->
     Sbom#{ ~"name" => Name}.
 
 fix_creators_tooling(Tool, #{ ~"creationInfo" := #{~"creators" := [ORT | _]}=Creators}=Sbom) ->
-    SHA = list_to_binary(string:trim(".sha." ++ os:cmd("git rev-parse HEAD"))),
-    Sbom#{~"creationInfo" := Creators#{ ~"creators" := [ORT, <<Tool/binary, SHA/binary>>]}}.
+    %% TODO: we do not always run inside a git repo. fix me.
```

Comment:
It should be simple to fix so that we are always running side a git repo if you want.

Reply:
fixed. Since we have access to the scan-results info, we can take the commit information from there, which makes `otp-compliance.es` to use the same commit as the last commit in the repo.